data/curators-whitelist.json

```
@@ -149,5 +149,38 @@
         "0x54f90Cbed0ba50fE7eE6113d8651eBdbF35938e2"
       ]
     }
+  },
+  {
+    "image": "",
+    "name": "Swissborg",
+    "url": "",
+    "verified": true,
+    "addresses": {
+      "1": [
+        "0x259c444b50e3Ab173c4f850BB40d85A9EA0230F3",
+        "0xAC15982Ca8A8e8BAc738FE492b84D8761B4384a3"
+      ]
+    },
+    "hidden": true
+  },
```

```suggestion
  {
    "name": "Swissborg",
    "verified": true,
    "addresses": {
      "1": [
        "0x259c444b50e3Ab173c4f850BB40d85A9EA0230F3",
        "0xAC15982Ca8A8e8BAc738FE492b84D8761B4384a3"
      ]
    },
    "hidden": true
  },
```

`image` and `url` are optional so you better not provide anything